src/uiveri5.js

```diff
@@ -345,8 +345,10 @@ function run(config) {
 
       browser._loadUI5Dependencies = function () {
         return browser.executeAsyncScriptHandleErrors('loadUI5Dependencies', {
-          waitForUI5Timeout: waitForUI5Timeout,
-          waitForUI5PollingInterval: config.timeouts.waitForUI5PollingInterval,
+          autoWait: {
```

Comment:
and how exactly do I pass arbitrary config to autoWaiters, how would I disable promiseWaiter ?